### DISCUSSION TITLE
SSL Certificate Failed to Renew

### DISCUSSION THREAD

**Discussion by nbouvrette** [vercel/community#5884] [Help] [answered]
### Page to Investigate

https://www.avansai.com

### Steps to Reproduce

I just got this email:

> Hi Nicolas Bouvrette,
> 
> An SSL Certificate for [avansai.com](http://avansai.com/) failed to renew due to a CAA DNS record conflict.
> 
> To fix this, please remove the conflicting CAA DNS record from your domain. See [this guide](https://c.knock.app/t/eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpYXQiOjE3MDg0MjM1MzUzODg2MzcsImlkeCI6MCwibWlkIjoiMmNjdkFIUG52UnlDNW53TWxIWElYRll3N1NLIiwidXJsIjoiaHR0cHM6Ly92ZXJjZWwuY29tL2tub3dsZWRnZS9kb21haW4tbm90LWdlbmVyYXRpbmctc3NsLWNlcnRpZmljYXRlP3V0bV9tZWRpdW09ZW1haWwmdXRtX3NvdXJjZT1ub3RpZmljYXRpb25zIn0.DELVdbMVtqi913d5gQW9U26m2jRG6KR9L2m2gqeh51U) for additional healp in resolving your CAA conflict.
> 
> You will receive a maximum of four warning emails.
> 
> Thank you.

We have not changed any configuration on our DNS for a very long time... the only thing I could see is that we had 2 `CAA` records.

I just deleted the one we had for Amazon Cloudfront but I suspect this will end up causing other issues. Has there been changes in the SSL certificate Vercel process that creates this situation and how can I put back my other CAA record?

**Comment by swarnava** [comment, marked as answer]
Hi,

This was a known issue on our side that affected a small number of domains where we incorrectly detected a CAA issue when there was no error. 
The issue was noticed soon after release and the affected check rolled back, but some alerts had already been sent. 

The certificate will renew successfully and the error message has already been cleared from your project settings.

**Reply by nbouvrette** [reply]
Thank you, I will set back my certificate and let you know if something else doesn't work as expected.
